feature/settings/import/src/main/res/values/strings.xml

```diff
@@ -37,4 +37,6 @@
     <string name="settings_import_oauth_error_oauth_not_supported">OAuth 2.0 is currently not supported with this provider.</string>
     <string name="settings_import_oauth_error_browser_not_found">The app couldn\'t find a browser to use for granting access to your account.</string>
     <string name="settings_import_general_settings_item">General settings</string>
+    <string name="settings_import_action_okay">OK</string>
+    <string name="settings_import_action_cancel">Cancel</string>
```

Comment:
I think we should avoid duplicating such basic strings (by putting them in a common/base module). Having independent copies increases the chance that translators will accidentally use different translations for different instances of these strings.

Reply:
The current project structure is tightly coupled and has all sort of dependencies. It is time to loosen them for better maintainability. 

If translations are a concern, there is a dictionary with commonly used translations. I also see no benefit in putting more effort into this. We should instead provide screenshots for translators to let them pick the best suitable translation in that context.

Reply:
Telling translators to make sure they use the same strings in multiple places might make sense as an interim solution when it would be a lot of work to change the app to have a single source for that string. But that isn't the case. We have that single source right now.

Creating multiple copies of strings that we want to be the same makes no sense. Modules like this will always share some functionality or assets with other parts of the app via a base/common module. And that's where shared strings should go. Having multiple copies of the same thing is a maintenance burden. There might be situations where having multiple copies might make sense for other reasons. I don't think this is one of them.

Reply:
To be honest, for the lifetime of this project there are only 2 shared strings available, so I consider this a no requirement. I also would question that `OK` or `Cancel` are always the best suitable option.

If this is important for you, I could move the 2 string into a separate module.